documentation/library_guide/parallel_api/parallel_range_algorithms.rst

```diff
@@ -0,0 +1,96 @@
+Parallel Range Algorithms
+#########################
+
+C++20 introduces the `Ranges library <https://en.cppreference.com/w/cpp/ranges>`_ and
+`range algorithms <https://en.cppreference.com/w/cpp/algorithm/ranges>`_ as a modern paradigm for expressing
+generic operations on data sequences.
+
+|onedpl_long| (|onedpl_short|) extends it with *parallel range algorithms* which can be used with the standard range
+classes to leverage |onedpl_short| ability of parallel execution on both the host computer and data parallel devices.
+
+oneDPL only supports random access ranges, because they allow simultaneous constant-time access to elements
+at any position in the range. This enables efficient workload distribution among multiple threads or processing units,
+which is essential for achieving high performance in parallel execution.
+
+.. Note::
+
+  The use of parallel range algorithms requires C++20 and the C++ standard libraries coming with GCC 10 (or higher),
+  Clang 16 (or higher) and Microsoft* Visual Studio* 2019 16.10 (or higher).
+
+Supported Range Views
+---------------------
+
+`Views <https://en.cppreference.com/w/cpp/ranges/view>`_ are lightweight ranges typically used to describe
+data transformation pipelines. The C++20 standard defines two categories of standard range views: factories
+and adaptors. A range factory generates its data elements on access via an index or an iterator to the range.
+A range adaptor transforms its underlying data range(s) or view(s) into a new view with modified behavior.
+
+The following C++ standard random access adaptors and factories can be used with the |onedpl_short|
+parallel range algorithms:
+
+* ``std::ranges::views::all``: A range adaptor that returns a view that includes all elements of a range.
+* ``std::ranges::subrange``: A utility that produces a view from an iterator and a sentinel or from a range.
+* ``std::ranges::iota_view``: A range factory that generates a sequence of elements by repeatedly incrementing
+  an initial value.
+* ``std::ranges::single_view``: A view that contains exactly one element of a specified value.
+* ``std::ranges::transform_view``: A range adaptor that produces a view that applies a transformation to each element
+  of another view.
+* ``std::ranges::reverse_view``: A range adaptor that produces a reversed sequence of elements provided by another view.
+* ``std::ranges::take_view``: A range adaptor that produces a view of the first N elements from another view.
+* ``std::ranges::drop_view``: A range adaptor that produces a view excluding the first N elements from another view.
+
+Supported Algorithms
+--------------------
+
+The ``<oneapi/dpl/algorithm>`` header provides the following parallel range algorithms in the
+``namespace oneapi::dpl::ranges``.
+All algorithms support both standard aligned (host) and device execution policies.
```

Comment:
Please merge these two sentences into a single paragraph and replace the period after "...device execution policies." with a colon (:).

Reply:
Changed in a different way to match the guidelines.